packages/harmony/src/components/layout/Paper/Paper.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled'
+
+import { toCSSVariableName } from 'utils/styles'
+
+import type { PaperProps } from './types'
+
+/** Base layout component used as a building block for creating pages and other components. */
+export const Paper = styled.div(
```

Comment:
Would we want to do styled(Flex)? Or is that opening up too big of a change?

Reply:
yea styled flex or box would be useful for having the props